CHANGELOG

```diff
@@ -2,6 +2,9 @@ devel
 -----
 * Integrated libiresearch log topic properly into ArangoDB logging system
 
+* Fixed a sleeping barber in fuerte. Added TLA+ model to prove that there
```

Comment:
This misses the changes in the Maintenance code.

Reply:
Good point. Will add to CHANGELOG.